src/core/apOrFunc.js

```diff
@@ -2,8 +2,11 @@
 /** @author Ian Hofmann-Hicks (evil) */
 
 const isApplicative = require('./isApplicative')
+const isTypeRepOf = require('./isTypeRepOf')
 
 const apOrFunc = af => x =>
```

Comment:
Is there a value in separating out the parameters and not using curry? Should we set a standard for it?

Reply:
It is curried because of how it is used in context, if it was not curried it would look like this:
```js
return either(
  compose(x => apOrFunc(f, x), Either.Left),
  runSequence
)
```

instead of just:
```js
const af =
  apOrFunc(f)

return either(
  compose(af, Either.Left),
  runSequence
)
```
and because the SumTypes and List are the only things that really use that first argument, they will always be in some sort of composition on the Left (or empty List/Array) side. So IMO it makes sense to write to the expected interface.